src/style.css

```diff
@@ -0,0 +1,52 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 div:nth-child(1) {
```

Comment:
replace `div` with `.stars__star`, write all selectors from 0 to 5 through a comma and apply `background: url(images/star-active.svg) no-repeat`